Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```
@@ -49,6 +49,8 @@
     price: 800
   - type: ResearchClient
   - type: TechnologyDatabase
+  - type: Speech # Goobstation - Lathe message on recipes update
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)